lib/spack/docs/containers.rst

```diff
@@ -485,7 +485,7 @@ after a given stage of the build. Let's consider as an example the following str
 
 containing both the custom template extension and the environment manifest file. To use a custom
 template, the environment must register the directory containing it, and declare its use under the
-``container`` configuration:
+``container`` configuration.  A user must also activate the environment or specify ``-e </opt/environment>`` when calling containerize.
```

Comment:
Can we move this not to line 522? I was thinking of something similar to:
```
A user can then generate the Dockerfile with the following command:

.. code-block::

   spack -e /opt/environment containerize

Note that the environment must be active for the new template directory to be considered by Spack. 
The recipe that gets generated...
```

Reply:
Yep. I like that better. I'll fix it.